scripts/stats.js

```diff
@@ -0,0 +1,90 @@
+import * as Style from "../src/js/style.js";
+import config from "../src/config.js";
+import { Command } from "commander";
+
+const program = new Command();
+program
+  .option("-a, --all-layers", "summary layer stats")
+  .option("-c, --layer-count", "count number of layers")
+  .option("-s, --layer-size", "size of all layers")
+  .option("-l, --layer <layer id>", "stats about one layer")
+  .option("-loc, --locales <locale1 locale2...>", "language codes", ["mul"])
+  .option("-pp, --pretty", "pretty-print JSON output")
+  .option(
+    "-pg, --print-group <group prefix>",
+    "print a list of the layers in a group"
+  )
+  .option("-pl, --print-layer <layer id>", "print the JSON of a layer");
+program.parse(process.argv);
+
+let opts = program.opts();
+
```

Comment:
```suggestion

if (Object.keys(opts).length === 1) program.help();

```

Since `npm run stats` without options prints nothing, let's default to showing help.

Reply:
That works.  Thanks!  A little annoying that you have to know to add the "--" as in `npm run stats -- -a`